src/tests/functional/plugin/conformance/subgraphs_dumper/src/main.cpp

```diff
@@ -133,7 +133,14 @@ int main(int argc, char *argv[]) {
 
     std::vector<std::string> local_cache_dirs = CommonTestUtils::splitStringByDelimiter(FLAGS_local_cache);
     std::vector<std::string> dirs = CommonTestUtils::splitStringByDelimiter(FLAGS_input_folders);
-    auto models = findModelsInDirs(dirs);
+
+    std::vector<SubgraphsDumper::Model> models;
+    try {
+        models = findModelsInDirs(dirs);
+    } catch (std::runtime_error& e) {
+        showUsage();
```

Comment:
`showUsage` is extra line as for me. To print error message and return is enough

Reply:
I suggest to discuss this today.

I put it here because I think it is good practice when a console application shows small (1 screen) help (ideally - issue-specific) in case of wrong arguments passed.